src/components/InlineWidget/InlineWidget.tsx

```diff
@@ -38,6 +75,7 @@ export class InlineWidget extends React.Component<Props> {
         style={this.props.styles || defaultStyles}
         data-url={this.props.url}
```

Comment:
We can remove this `url` data attribute since we are setting the url using the `initInlineWidget` function.